### DISCUSSION TITLE
Embargo end date

### DISCUSSION THREAD

**Discussion by sujayane** [dandi/helpdesk#144] [Q&A] [answered]
Hi, 

I have the following metadata validation error even though I already un-embargoed my dataset. Where would I put an end date to an embargo? This is the dataset: https://dandiarchive.org/dandiset/000897

access: Value error, An embargo end date is required for NIH awards to be in compliance with NIH resource sharing policy.

Thanks for your help!

**Comment by kabilar** [comment]
Thanks for the report @sujayane.  I think that this error is occurring because the `Access Information` is still set to `dandi:EmbargoedAccess`, instead of `dandi:OpenAccess`.

Hi @waxlamp @jjnesbitt, would you be able to update the `Access Information` for this Dandiset to `OpenAccess`?  Thank you.

**Comment by jjnesbitt** [comment, marked as answer]
Hi all, this has been addressed. The above mentioned dandiset seems to now show as valid.

@sujayane I'd suggest that you and anyone else who may be updating information about the dandiset refresh the client page in their browser before doing any more work. We're currently investigating an issue that may have resulted in your issue, where "stale" data from _before_ the un-embargo is used to update a dandiset.

**Reply by kabilar** [reply]
Thank you @jjnesbitt.

**Reply by sujayane** [reply]
Thank you everyone for such a quick response!
